a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -561,9 +561,12 @@ <h3>Introduction to key accessibility information</h3>
 					shown in the examples.</p>
 			</div>
 		</section>
+		            <section id="ways-of-reading">
+                <h3 data-localization-id="ways-of-reading-title">Ways of reading</h3>
+
 		<section id="visual-adjustments">
-			<h3 data-localization-id="visual-adjustments-title">Visual
-				adjustments</h3>
+			<h4 data-localization-id="visual-adjustments-title">Visual
```

Comment:
ID should be "ways-of-reading-visual-adjustments-title"
to mirror what we did for "additional-accessibility-information-clarity-title"